DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/CTF.h

```diff
@@ -43,12 +43,14 @@ struct CompressedDigits {
   std::vector<uint16_t> bcInc;    // increment in BC if the same orbit, otherwise abs bc
   std::vector<uint32_t> orbitInc; // increment in orbit
   std::vector<uint8_t> nChan;     // number of fired channels
+  // std::vector<uint8_t> eventFlags;     // special flags about event conditions: pile-up, not use for collision time, not use for event plane, etc.
 
   // channel data
   std::vector<uint8_t> idChan;   // channels ID: 1st on absolute, then increment
   std::vector<int16_t> cfdTime;  // CFD time
-  std::vector<uint32_t> qtcAmpl; // Amplitude
-  std::vector<uint8_t> qtc;      // QTC
+  std::vector<int32_t> qtcAmpl;  // Amplitude
```

Comment:
you wrote me that the range of the amplitude is 0 : 4096, why do you need signed int32? Actually, would not uint16_t suffice?